internal/cmd/build/main.go

```diff
@@ -187,6 +188,7 @@ func (b *builder) integrationTest() error {
 	cmd := b.cmdFromRoot(args...)
 	cmd.Dir = filepath.Join(cmd.Dir, "test")
 	cmd.Env = append(os.Environ(), "DISABLE_SERVER_1_25_TESTS=1")
+	cmd.Env = append(os.Environ(), "ENABLE_SERVER_1_27_TESTS=1")
```

Comment:
This PR https://github.com/temporalio/sdk-go/pull/1832/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR92 also adds a flag for the same purpose can we standardized on this approach?

Reply:
I will go with @antlai-temporal's approach then.